src/app/components/header/logo/logo.component.html

```diff
@@ -0,0 +1,12 @@
+<a href="#" class="link">
+    <img
+      ngSrc="../../../assets/images/logo-mobile.png"
+      srcset="
+        ../../../assets/images/logo-mobile.png   480w,
+        ../../../assets/images/logo-tablet.png   768w,
+        ../../../assets/images/logo-desktop.png 1200w
```

Comment:
Using srcset is just great!
One thing that can be added to improve it is the sizes attribute, like this  **sizes="(max-width: 480px) 480px, (max-width: 768px) 768px, 1200px"**
You can read more about it here [image-directive](https://angular.io/guide/image-directive)

Reply:
Thanks!